src/Library/demos/View Switcher/code.rs

```diff
@@ -0,0 +1,47 @@
+use crate::workbench;
+use adw::prelude::*;
+
+use crate::glib::clone;
+use gtk::glib;
+use std::cell::Cell;
+use std::rc::Rc;
+
+pub fn main() {
+    let notifications_page: adw::ViewStackPage = workbench::builder().object("page3").unwrap();
+    let notification_list: gtk::ListBox = workbench::builder().object("notification_list").unwrap();
+
+    let notification_count = Rc::new(Cell::new(5));
```

Comment:
That is a 1:1 port of JavaScript, which is good.
Tbh I think the JavaScript version is a bit odd, and it especially shows with the Rust version.
Let's make this a normal integer instead.